.changeset/spicy-seas-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+fix: deployed worker unable to invoke itself in memory queue
```

Comment:
can you expand and add details on what users need to do (i.e. add `NEXT_CACHE_REVALIDATION_WORKER`).
Thanks!